src/routes/(auth)/sign-in/+page.svelte

```diff
@@ -2,16 +2,33 @@
   import { applyAction, enhance } from '$app/forms';
   import Logo from '$lib/Logo.svelte';
   import { toast } from 'svelte-sonner';
+  import { ArrowLeft } from 'lucide-svelte';
```

Comment:
the Icon library we are using is `@iconify/svelte`.
to add a new Icon just do this:
```svelte
<script>
import Icon from "@iconify/svelte"
</script>

<a href="/" class="flex items-center gap-2">
    <Icon icon="icon-name" class="text-2xl" />
    <span class="text-sm">Back</span>
</a>

```